src/OrchardCore/OrchardCore.ContentManagement.GraphQL/Queries/Types/DynamicContentFieldsIndexAliasProvider.cs

```diff
@@ -57,13 +57,13 @@ public IEnumerable<IndexAlias> GetAliases()
                         continue;
                     }
 
-                    var (index, indexType) = fieldProvider.GetFieldIndex(field);
+                    var fieldIndex = fieldProvider.GetFieldIndex(field);
```

Comment:
Here should check if it's null.

```suggestion
                    var fieldIndex = fieldProvider.GetFieldIndex(field);
                    if(fieldIndex is null)
                    {
                        continue;
                    }
```